LibVLCSharp.Uno/SuspensionManager.cs

```diff
@@ -0,0 +1,52 @@
+﻿using LibVLCSharp.Shared;
+using Windows.Storage;
+
+namespace LibVLCSharp.Uno
+{
+    /// <summary>
+    /// Suspension manager
+    /// </summary>
+    public static class SuspensionManager
```

Comment:
I tend to call my classes that contains only static functions "helpers", so `SuspensionHelpers` ?